modules/account-lib/package.json

```diff
@@ -53,6 +53,7 @@
     "ethereumjs-util": "5.2.0",
     "libsodium-wrappers": "^0.7.6",
     "lodash": "^4.17.15",
+    "long": "^4.0.0",
```

Comment:
Was this just totally undeclared here and brought in via a transitive dependency? :fearful:

Reply:
looks like brought by changes made in `transaction.ts`

Reply:
I think so. I saw we were using it transactions.ts file that Barath mentioned, but when I tried to run the test locally they were failing due to that missing dependency.

Reply:
fixed